.github/workflows/build-test-release.yml

```diff
@@ -557,3 +557,37 @@ jobs:
           artifacts: packages/*
           omitBody: true
           replacesArtifacts: true
+
+  publish-kube-rbac-proxy-image-marketplace-owned-ecr:
+    runs-on: ubuntu-latest
+    needs:
+      - validate-release-increment
+    if: ${{ needs.validate-release-increment.outputs.perform-release == 'true' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Install tools
+        run: |
+          make install-tools
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR Private
+        id: login-ecr-private
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Build and push container image to Amazon ECR
+        env:
+          DOCKER_REPO: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lumigo/lumigo-kubernetes-rbac-proxy
+          VERSION: ${{ needs.validate-release-increment.outputs.version }}
+        run: |
+          aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 709825985650.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
Why repeating `709825985650.dkr.ecr.us-east-1.amazonaws.com`. It feels like you should pipe in the first segment of `$DOCKER_REPO`